backend/src/schema/resolvers/reports.js

```diff
@@ -5,31 +5,31 @@ export default {
       const { resourceId, reasonCategory, reasonDescription } = params
       const { driver, user } = context
       const session = driver.session()
-      const writeTxResultPromise = session.writeTransaction(async txc => {
-        const reportRelationshipTransactionResponse = await txc.run(
-          `
+      try {
```

Comment:
why are you wrapping this in a `try`... `write/read` transaction are `promises`, so they shouldn't need to be wrapped in `try`, oder?